### DISCUSSION TITLE
AStyle, seriously?

### DISCUSSION THREAD

**Discussion by ghost** [RolandHughes/copperspice#7] [General]
One of the very few first commits of you is adding an astyle rc file. The last time I hear about astyle is very long ago. It can't handle modern C++ syntax. People all use clang-format nowadays. Adding a .clang-format file is more logical.

**Comment by RolandHughes** [comment]
It's never failed me with C++17

clang has a lot of issues. When QtCreator started using it the IDE became unusable on most systems because it was sooooooo slooooow
